src/commands/index.ts

```diff
@@ -654,6 +654,27 @@ export class VSCodeCommands implements Disposable {
           },
         );
       }),
+      commands.registerCommand(
+        "dbtPowerUser.showDocumentation",
+        async (modelName) => {
+          const result = queryManifestService.getEventByCurrentProject();
+          if (!result) {
+            return;
+          }
+          const { event } = result;
+          if (!event) {
+            return;
+          }
+          const { nodeMetaMap } = event;
+          const model = nodeMetaMap.get(modelName);
+          if (!model?.path) {
+            return;
+          }
+          const doc = await workspace.openTextDocument(Uri.file(model.path));
```

Comment:
Does this reopen the existing doc if it is already open in the editor area?

Reply:
Is the path absolute or relative?

Reply:
its absolute path. yeah. it opens existing tab for file